proto/build.rs

```diff
@@ -29,7 +29,7 @@ fn main() {
                 if !dent.file_type().is_file() {
                     return None;
                 }
-                Some(format!("{}", dent.path().display()))
+                Some(format!("{}", dent.path().display()).replace('\\', "/"))
```

Comment:
Could you add a comment please? This is a very mysterious change unless you know the reason why